client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -0,0 +1,240 @@
+<div data-ng-controller="EmailsMigrateAccountCtrl as ctrl">
+    <div data-wizard
+        data-wizard-bread-crumb
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="migrateAccount"
+        data-wizard-title="i18n.email_tab_modal_migrate_title">
+
+        <!-- Step 1 -->
+        <div data-wizard-step
+            data-wizard-step-on-load="getServiceTypes"
+            data-wizard-step-valid="ctrl.migrateDestinationForm.$valid && ctrl.migrate.destinationEmail">
+
+            <div data-ng-if="!ctrl.loading || ctrl.availableServices">
+                <form class="form-horizontal" name="ctrl.migrateDestinationForm" novalidate>
+
+                    <!-- No Service -->
+                    <div data-ng-if="!ctrl.availableServices.length">
+                        <p data-i18n-static="email_tab_modal_migrate_no_email_service"></p>
+                        <p><a href="https://www.ovh.com/emails/" target="_blank" class="btn btn-primary" data-i18n-static="email_tab_modal_migrate_order_email_service"></a></p>
+                    </div>
+                    <!-- /No Service -->
+
+                    <!-- Has Services -->
+                    <div data-ng-if="ctrl.availableServices.length">
+                        <p data-i18n-static="email_tab_modal_migrate_select_platform_text"></p>
+                        <p>
+                            <small class="text-danger">*</small>
+                            <small data-i18n-static="required_fields"></small>
+                        </p>
+
+                        <!-- Service Type -->
+                        <div class="form-group">
+                            <label class="control-label col-md-4 required" for="migrateServiceType" data-i18n-static="email_tab_modal_migrate_service_type_label"></label>
+                            <div class="col-md-8">
+
+                                <!-- One Service -->
+                                <p class="form-control-static"
+                                    data-ng-if="ctrl.availableServices.length === 1"
+                                    data-ng-bind="ctrl.migrate.serviceType"></p>
+                                <!-- /One Service -->
+
+                                <!-- Multiple Services -->
+                                <div class="radio-inline"
+                                    data-ng-if="ctrl.availableServices.length > 1"
+                                    data-ng-repeat="serviceType in ctrl.availableServices">
+                                    <label>
+                                        <input id="migrateServiceType{{$index}}" type="radio" name="serviceType"
+                                            data-ng-model="ctrl.migrate.serviceType"
+                                            data-ng-value="serviceType"> {{::serviceType}}
```

Comment:
Prefer `ng-bind`?

Reply:
Why it's preferred ? I'll have to add a useless span for the same goal